bin/git-sync-mirror.sh

```diff
@@ -180,15 +180,22 @@ if ! whoami &> /dev/null; then
 fi
 
 # Create local repositories
-LOCAL_REPO_SRC="$(mktemp -d)"
-LOCAL_REPO_DST="$(mktemp -d)"
+LOCAL_REPO_SRC="${LOCAL_REPO_SRC:-"$(mktemp -d)"}"
+LOCAL_REPO_DST="${LOCAL_REPO_DST:-"$(mktemp -d)"}"
+
+if [ -z "$(ls -A ${LOCAL_REPO_SRC})" ]; then
+     clone_local_repo "${SRC_REPO}" "${LOCAL_REPO_SRC}"
+fi
+
+if [ -z "$(ls -A ${LOCAL_REPO_DST})" ]; then
```

Comment:
see my previous comment.
```suggestion
if [ ! -d "${LOCAL_REPO_DST}/.git" ]; then
```